src/EnergyPlus/HVACInterfaceManager.cc

```diff
@@ -147,34 +146,49 @@ namespace HVACInterfaceManager {
 		// from the outlet of one side of the loop get transfered directly
 		// to the inlet node of the corresponding other side of the loop.
 
-		// REFERENCES:
-		// na
-
-		// Using/Aliasing
 		using DataLoopNode::Node;
 		using namespace DataConvergParams;
 		using DataContaminantBalance::Contaminant;
 
-		// Locals
-		// SUBROUTINE ARGUMENT DEFINITIONS:
-
-		// SUBROUTINE PARAMETER DEFINITIONS:
-
-		// INTERFACE BLOCK SPECIFICATIONS
-		// na
-
-		// DERIVED TYPE DEFINITIONS
-		// na
-
-		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 		static Array1D< Real64 > TmpRealARR( ConvergLogStackDepth ); //Tuned Made static
 		Real64 DeltaEnergy;
-		// FLOW:
+
+		if ( ( CalledFrom == CalledFromAirSystemDemandSide ) && ( OutletNode == 0 ) ) {
+			// Air loop has no return path - only check mass flow and then set return inlet node mass flow to sum of demand side inlet nodes
+			AirLoopConvergence( AirLoopNum ).HVACMassFlowNotConverged( 1 ) = false;
+			AirLoopConvergence( AirLoopNum ).HVACHumRatNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACTempNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACEnergyNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACEnthalpyNotConverged( 1 ) = true;
+			AirLoopConvergence( AirLoopNum ).HVACPressureNotConverged( 1 ) = true;
+
+			int totDemandSideMassFlow = 0.0;
+			int totDemandSideMinAvail = 0.0;
+			int totDemanSideMaxAvail = 0.0;
```

Comment:
@rraustad Notice anything funny about these three lines?  Couldn't figure out why totDemandSideMassFlow was always zero even though Node( demInNode).MassFlowRate was not . . . doh!